client/OPUNetTransportLayer.cpp

```diff
@@ -1146,7 +1146,7 @@ bool OPUNetTransportLayer::DoImmediateProcessing(Packet &packet, sockaddr_in &fr
 			if (tlMessage.searchQuery.gameIdentifier != gameIdentifier) {
 				return true;		// Packet handled (discard)
 			}
-			// Verify password
+			// Verify joinRequestPassword
 			if (strncmp(tlMessage.searchQuery.password, password, sizeof(password)) != 0) {

```

Comment:
Is this part correct? It seems odd the comment was updated, but the field names were not.